app/assets/stylesheets/frontend/helpers/_organisations.scss

```diff
@@ -936,6 +936,10 @@ img.organisation-logo-custom {
   @extend .organisation-logo-identity-medium;
   border-left: none;
 }
+.organisation-logo-stacked-custom-medium {
```

Comment:
They could be. I was just matching the style of all the other logos where they are split into two different blocks.